packages/react-components/react-nav-preview/stories/NavDrawerOverlay/NavDrawerOverlay.stories.tsx

```diff
@@ -3,38 +3,34 @@ import {
   NavCategory,
   NavCategoryItem,
   NavDrawer,
-  NavDrawerProps,
   NavItem,
   NavSubItem,
   NavSubItemGroup,
 } from '@fluentui/react-nav-preview';
 import { DrawerBody } from '@fluentui/react-drawer';
-import { makeStyles, shorthands, tokens } from '@fluentui/react-components';
 import { Folder20Filled, Folder20Regular, bundleIcon } from '@fluentui/react-icons';
-const useStyles = makeStyles({
-  root: {
-    ...shorthands.overflow('hidden'),
-    display: 'flex',
-    height: '480px', // arbitrary value, for demo purposes only
-    // arbitrary value, for dramatic effect only
-    // I know this is ugly and wrong, but styling is coming, I promise.
-    backgroundColor: tokens.colorBackgroundOverlay,
-  },
-});
+// eslint-disable-next-line @fluentui/no-restricted-imports
+import { useIsSSR } from '@fluentui/react-utilities';
 
 const Folder = bundleIcon(Folder20Filled, Folder20Regular);
 
-export const Default = (props: Partial<NavDrawerProps>) => {
-  const styles = useStyles();
-
+export const NavDrawerOverlay = () => {
+  const isSSR = useIsSSR();
+  const [open, setOpen] = React.useState(true);
+  React.useEffect(() => {
+    if (!isSSR) {
+      setOpen(true);
+    }
+  }, [isSSR]);
   const someClickHandler = () => {
     console.log('someClickHandler');
   };
```

Comment:
We talked offline about this, but I've pulled this PR and adding a button to trigger the drawer does work. This should remove the need for useIsSSR.

One thing to note as well is that I don't think we should use `useIsSSR` inside stories. This is test specific logic that should only be done within the test. Therefore Before this PR goes in we should remove this. Since we are covering this behind a button (if no other issues come up), we don't have to disable the story from `test-ssr` anymore.

Reply:
ten four - removed!